roles/nat64_appliance/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure nat64-appliance directories
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    mode: '0755'
+  loop:
+    - "{{ cifmw_nat64_appliance_basedir }}"
+    - "{{ cifmw_nat64_appliance_workdir }}"
+    - "{{ cifmw_nat64_appliance_workdir }}/artifacts"
```

Comment:
you can loop only on `artifacts` and `logs`  - ansible will create the parent directories in that case.
You can therefore run this:
```YAML
- name: Ensure needed directories exist
  ansible.builtin.file:
    path: "{{ cifmw_nat64_appliance_workdir }}/{{ item }}"
    stage: directory
    mode: "0755"
  loop:
    - artifacts
    - logs
```

Reply:
Done